app.py

```diff
@@ -1,37 +1,47 @@
-from flask import Flask, request, render_template_string, redirect, url_for
+from flask import Flask, request, render_template, redirect, url_for
 import json
 import os 
 app= Flask(__name__)
```

Comment:
Randuri libere + spatii

```suggestion
from flask import Flask, request, render_template, redirect, url_for
import json
import os

app = Flask(__name__)
```